src/os/ui/filter/op/op.js

```diff
@@ -283,3 +295,21 @@ os.ui.filter.op.Op.prototype.validate = function(value, key) {
 
   return false;
 };
+
+
+/**
+ * Text reused in several ops; to save on total the KB of the min file
+ * @enum {string}
+ * @const
+ */
+os.ui.filter.op.Op.TEXT = {
+  CASE_INSENSITIVE: '  (INFO: not case-sensitive)',
+  CASE_INSENSITIVE_DETAIL: '<strong>Case-insensitive</strong><sup>1</sup>, i.e. "A" == "a" and<br />' +
+  '<strong>Wildcard</strong> support, i.e. "a*" == "ab"<br /><br /><sup>1</sup>&nbsp;Typically, case' +
+  '-<i>in</i>sensitive text search is available for "like" filters. Some OGC Servers may not implement it.',
```

Comment:
The reference to OGC servers here is not totally generic. This filter builder can be used for other server types. "Data Providers may not support it" would be a more generic description.

Reply:
+1